WalletWasabi/Crypto/ZeroKnowledge/LinearRelation/Knowledge.cs

```diff
@@ -1,37 +0,0 @@
-using NBitcoin.Secp256k1;
-using System.Collections.Generic;
-using System.Linq;
-using WalletWasabi.Helpers;
-
-namespace WalletWasabi.Crypto.ZeroKnowledge.LinearRelation
-{
-	public class Knowledge
```

Comment:
i meant to bring it up for discussion, not just do it yet, there are some downsides to this too and i'm not sure it's a good idea yet

Reply:
Reverted.